symphony/app/fbcnms-projects/workflows/src/proxy/proxy.js

```diff
@@ -13,77 +13,92 @@ import httpProxy from 'http-proxy';
 import logging from '@fbcnms/logging';
 import transformerRegistry from './transformer-registry';
 import {getTenantId} from './utils.js';
+import type {
+  ExpressRouter,
+  ProxyCallback,
+  ProxyNext,
+  ProxyRequest,
+  ProxyResponse,
+  TransformerEntry,
+} from '../types';
 
 const logger = logging.getLogger(module);
 const router = Router();
 router.use(bodyParser.urlencoded({extended: false}));
 router.use('/', bodyParser.json());
 
-export default async function(proxyTarget) {
-  const transformers = await transformerRegistry({proxyTarget});
+export default async function(proxyTarget: string) {
+  const transformers: Array<TransformerEntry> = await transformerRegistry({
```

Comment:
This type annotation isn't necessary. transformerRegistry is already typed, so flow can infer that transformers is an `Array<TransformerEntry>`. If you remove this type you'll notice the flow coverage doesn't decrease - it's still strictly typed. Avoiding unnecessary typing is cleaner. The flow plugin can help with this.

Reply:
done